roles/cinder-control/templates/etc/cinder/cinder.encryption.conf

```diff
@@ -0,0 +1,8 @@
+{% if cinder.encryption.enabled|bool and barbican.enabled|default(false)|bool %}
+[key_manager]
+api_class = 'castellan.key_manager.barbican_key_manager.BarbicanKeyManager'
```

Comment:
nit - the single quotation marks are not necessary https://docs.openstack.org/newton/config-reference/block-storage/volume-encryption.html